pkg/services/serviceaccounts/database/store.go

```diff
@@ -397,7 +434,7 @@ func (s *ServiceAccountsStoreImpl) MigrateApiKeysToServiceAccounts(ctx context.C
 
 	if len(basicKeys) > 0 {
 		for _, key := range basicKeys {
-			err := s.CreateServiceAccountFromApikey(ctx, key)
+			_, err := s.CreateServiceAccountFromApikey(ctx, key)
 			if err != nil {
 				s.log.Error("Migating to service accounts failed with error", err.Error())
```

Comment:
```suggestion
				s.log.Error("Migrating to service accounts failed with error", err.Error())
```

Reply:
I'm not sure if this typo is not from upstream, so up to you to fix it or not :)

Reply:
No. This is completely my typo. I will fix that.